src/app/docs/toolbar/toolbar-docs.component.html

```diff
@@ -0,0 +1,156 @@
+<sky-docs-demo-page
+  moduleName="SkyToolbarModule"
+  moduleSourceCodePath="src/app/public/modules/toolbar/"
+  pageTitle="Toolbar"
+>
+  <sky-docs-demo-page-summary>
+    The toolbar component displays actions for lists, reocrds, and tiles.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
```

Comment:
We also need `headingStyle="success"` so that the green checkmark appears in the header.